paper-radio-group.html

```diff
@@ -71,116 +69,85 @@
 
     properties: {
       /**
-       * Fired when the selected element changes to user interaction.
+       * If you want to use the attribute value of an element for `selected` instead of the index,
+       * set this to the name of the attribute.
        *
-       * @event paper-radio-group-changed
+       * @attribute attrForSelected
+       * @type {string}
        */
-
-      /**
-       * Gets or sets the selected element. Use the `name` attribute of the
-       * <paper-radio-button> that should be selected.
-       *
-       * @attribute selected
-       * @type String
-       * @default null
-       */
-
-      selected: {
+      attrForSelected: {
         type: String,
-        value: null,
-        notify: true,
-        reflectToAttribute: true,
-        observer: "_selectedChanged"
+        value: "name",
+        readOnly: true
       }
     },
 
     keyBindings: {
-      'left up': '_selectPrevious',
-      'right down': '_selectNext',
+      'left up': 'selectPrevious',
+      'right down': 'selectNext',
     },
 
-    _selectedChanged: function() {
+    ready: function() {
       // TODO: This only needs to be async while a domReady event is unavailable.
-      this.async(function() {
-        this._selectIndex(this._valueToIndex(this.items, this.selected));
-        this.fire('paper-radio-group-changed');
-      });
+      if (this.selected) {
+        this.async(function() {
+          this._valueToItem(this.selected).checked = true;
```

Comment:
Like, not in an `async` at all? No, because the selected observer would still fire and be run before the `paper-radio-button` initializers are run